### PR TITLE
New package: chaco - a graph partitioning library

### PR DESCRIPTION
**Pull request by pbrady**

### CODE REVIEW

File: var/spack/repos/builtin/packages/chaco/package.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class Chaco(CMakePackage):
+    """Graph partitioning library"""
+
+    homepage = "https://gitlab.com/truchas/tpl-forks/chaco"
+    git      = "https://gitlab.com/truchas/tpl-forks/chaco.git"
+
+    maintainers = ['pbrady']
+
+    version('develop', branch="truchas")
+    version('develop-2020-07-16', commit='92a877b381933d12b02507413897f696d81b4682', preferred=True)
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/chaco/package.py]:
```suggestion
    version('2020-07-16', commit='92a877b381933d12b02507413897f696d81b4682', preferred=True)
```

**Review by alalazo — CHANGES_REQUESTED** [review]
Same as question as for charrapal on whether this fork is the only one currently used, but basically LGTM.

**Comment by pbrady** [comment]
Unlike chaparral they are tarballs of chaco floating around but there is no "official" repo.  The closest we've found is the chaco subpackage of trilinos: https://github.com/trilinos/Trilinos/tree/64d50199f342860b00d16cb78cb5730de66438b9/packages/seacas/libraries/chaco
but that would take some work to extract and build as a separate package.

**Comment by alalazo** [comment]
I'd say let's stick with the same strategy as charrapal and keep this name for the time being. Also, can you rebase to run Gitlab pipelines on a new merge commit?

**Comment by pbrady** [comment]
rebased

**Review by alalazo — APPROVED** [review]

**Pull request merged**